src/overlays/actors/ovl_Arrow_Fire/z_arrow_fire.c

```diff
@@ -231,7 +234,7 @@ void ArrowFire_Draw(Actor* thisx, GlobalContext* globalCtx2) {
         } else {
             Matrix_Translate(0.0f, 1500.0f, 0.0f, MTXMODE_APPLY);
         }
-        Matrix_Scale(this->radius * 0.2f, this->unk_158 * 4.0f, this->radius * 0.2f, MTXMODE_APPLY);
+        Matrix_Scale(this->radius * 0.2f, this->trailLen * 4.0f, this->radius * 0.2f, MTXMODE_APPLY);
```

Comment:
Is this really a "trail" length? In my understanding it stretches the fire halo in the direction of shooting the arrow, and calling that a trail may bring confusion with the arrow's own trail (the white line, iirc)